timeunit_string.go

```diff
@@ -0,0 +1,28 @@
+// Code generated by "stringer -type=timeUnit"; DO NOT EDIT.
```

Comment:
This is good however I don't think we use the time unit string representations anywhere... should we still keep this file?

Reply:
If we don't, then we should probably remove the go generate bit from the gocron file, because I don't believe its needed for iota the timeunits

Reply:
I say we remove it, then; agree?